platform/firecamp-ui/src/components/dropdown/Dropdown.stories.tsx

```diff
@@ -86,7 +86,7 @@ const Template = (args) => {
           }]
         }
         onSelect={item => {
-          setSelected(item.name || 'oopss...')
+          setSelected(item.name || 'oops...')
```

Comment:
some projects are intentionally cute. this one doesn't really appear to be.

Reply:
Ha ha... I am trying to make it cute. :D

Reply:
Happy to drop if you like...